### ISSUE TITLE
COPY / ADD file in script

### ISSUE THREAD

**Issue by galderz** [JakeWharton/dockerfile-shebang#15]
Say you have a dockerfile script that uses another file? `COPY <filename>` does not seem to work. How would you handle this?

**Comment by galderz** [comment]
One option is of course to inline the contents of the files into the dockerfile, but that can be quite tedious. It'd be a pain with binary files. Maybe a volume it's the easiest?

**Comment by JakeWharton** [comment]
Yeah I think this is one of the biggest limitations that there's no good answer for. Or at least I don't have one and would love to be corrected on it. I suspect this tool simply cannot accommodate all styles of Dockerfile.
